src/main/java/com/bootcamp/demo/model/Booking.java

```diff
@@ -0,0 +1,104 @@
+package com.bootcamp.demo.model;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class Booking {
+    private String id;
+    private String serialNumber;
+    private long accountId;
+    private String startDate;
+    private String endDate;
```

Comment:
Since these were saved as String, you could have in this class the methods which convert from String -> LocalDateTime, in order to have these in 1 place & not need to duplicate them. Also, as such, the class would be the one responsible for doing the conversion as it is fit (e.g. using always the same DateTimeFormatter).

Reply:
When resolving a comment, if the suggestion is not implemented, it is usually good to outline why not, in order to discuss & reach an agreement with the other members of the team regarding what is best.